README.md

```diff
@@ -38,6 +38,11 @@ npm i
 npm run dev-server
 ```
 
+### Configuration
+
+rocket! uses the `dotenv` to configure environment variables and requires the following in the .env file located in the root of the project directory:
+- NODE_ENV=development
+
 ### Roadmap
 
 View the project roadmap [here](LINK_TO_DOC)
```

Comment:
Not really part of your PR but would you be able to add a link to our trello in the parenthesis so we have a reference to it? One of us can do it inline or you can add it and push it up. Other than that, looks great!